cicd/3-app/deploy-development-stack.rb

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'fileutils'
+require 'open3'
+require 'erb'
+require 'json'
+
+# Default options
+options = {
+  profile: 'codeorg-dev',
+  region: 'us-east-1',
+  stack_name: 'javabuilder-dev',
+  base_domain_name: 'dev-code.org',
+  subdomain_name: 'javabuilder-dev',
+  hosted_zone_id: 'Z2LCOI49SCXUGU',
```

Comment:
In our AWS Dev account the Hosted Zone ID for `dev-code.org` is `Z07248463JGJ44FME5BZ5`